js/src/lib.rs

```diff
@@ -107,6 +107,26 @@ impl JsHandle {
         panic!("unsupported doughnut version");
     }
 
+    /// Sign and return sr25519 signature
+    // #[wasm_bindgen(method, js_class = "Doughnut", js_name = signSr25519)]
```

Comment:
elsewhere we are renaming the rust function like
```
#[allow(non_snake_case)]
pub fn signSr25519
```
to get the same result.
Would prefer to stay consistent